includes/Traits/Decoder.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Trait Decoder
+ *
+ * @package   Google\Web_Stories\Traits
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Traits;
+
+/**
+ * Trait Decoder
+ *
+ * @package Google\Web_Stories\Traits
+ */
+trait Decoder {
+	/**
+	 * Decodes string if encoded.
+	 *
+	 * @param string $string String to decode.
+	 * @return string Decoded string.
+	 */
+	protected function base64_decode( $string ) {
+		if ( 0 === strpos( $string, '__WEB_STORIES_ENCODED__' ) ) {
+			$string = str_replace( '__WEB_STORIES_ENCODED__', '', $string );
+			return mb_convert_encoding( base64_decode( $string ), 'UTF-8', 'UTF-16LE' );
```

Comment:
Don't we need to check if `mb_convert_encoding` exists like this. 
https://github.com/google/web-stories-wp/blob/df921610c406898a98fa4112f1c0eb44f37b345b/includes/Story_Renderer/HTML.php#L152

This makes this a requirement for https://github.com/google/web-stories-wp/issues/4791

Reply:
Yep, needs a check, good catch.

At the same time, wondering if it would make sense to use a polyfill like https://github.com/symfony/polyfill-mbstring...